src/Broadway/EventStore/InMemoryEventStore.php

```diff
@@ -46,25 +48,35 @@ public function append($id, DomainEventStreamInterface $eventStream)
     {
         $id = (string) $id;
 
-        if (! isset($this->events[$id])) {
-            $this->events[$id] = [];
+        $aggregateEvents = [];
+        if (isset($this->events[$id])) {
+            $aggregateEvents = $this->events[$id];
         }
 
         foreach ($eventStream as $event) {
-            $playhead = $event->getPlayhead();
-            $this->assertPlayhead($this->events[$id], $playhead);
-
-            $this->events[$id][$playhead] = $event;
+            $aggregateEvents = $this->insertEvent($aggregateEvents, $event, $eventStream);
         }
+
+        $this->events[$id] = $aggregateEvents;
```

Comment:
Sure, it is a bug that is already present.

When providing an empty event stream, it will initialise `$this->events` with an empty array. This will cause the isset in `load` to succeed, which on its turn will return an empty event stream. So unless you guard for that in your aggregate, you will end up with an empty (and thus invalid)  aggregate. 
I think `load` should also check for array count instead of just isset.